libs/langchain/langchain/evaluation/agents/trajectory_eval_chain.py

```diff
@@ -281,7 +281,7 @@ def output_keys(self) -> List[str]:
 
     def prep_inputs(self, inputs: Union[Dict[str, Any], Any]) -> Dict[str, str]:
         """Validate and prep inputs."""
-        if "reference" not in inputs:
+        if "reference" in inputs:
```

Comment:
I think original code is correct @baskaryan, but not 100% sure. Also I think the "if in" statement could be unsafe since inputs can be of Any type

Reply:
Part of the motivation could be that `_format_reference` takes a str or None, and this is the only place it's called, so currently it's only ever called with None.

We could potentially remove the conditional altogether, as an alternative.